listener/listener.go

```diff
@@ -172,7 +189,7 @@ func setSubscriptionEntity(ctx context.Context, l *Listener) error {
 	if l.subscriptionName == "" {
 		l.subscriptionName = defaultSubscriptionName
 	}
-	subscriptionEntity, err := getSubscriptionEntity(ctx, l.subscriptionName, l.namespace, l.topicEntity)
+	subscriptionEntity, err := l.getSubscriptionEntity(ctx, l.subscriptionName, l.namespace, l.topicEntity)
```

Comment:
I noticed in both of the calls to this function l.namespace, and l.topicEntity were passes in. Do you expect the values will always come from the listener when calling this function, if so seems like could be retrieved from the listener struct it is being called on

Reply:
yep changed that.